salt/beacons/bonjour_announce.py

```diff
@@ -188,12 +190,12 @@ def beacon(config):
                     grain_value = ','.join(grain_value)
             txt[item] = _enforce_txt_record_maxlen(item, grain_value)
             if LAST_GRAINS and (LAST_GRAINS.get(grain, '') != __grains__.get(grain, '')):
-                changes[str('txt.' + item)] = txt[item]
+                changes[six.text_type('txt.' + item)] = txt[item]
```

Comment:
Same recommendation as [here](https://github.com/saltstack/salt/pull/45238/files#diff-024253fb5f90e110e541d733210f45c1R199) for this line and line [198](https://github.com/saltstack/salt/pull/45238/files#diff-7d68acc24ab0070cad6e97c6aab3536fR198).